cmd/posix.go

```diff
@@ -421,7 +424,7 @@ func (s *posix) diskUsage(doneCh chan struct{}) {
 // Make a volume entry.
 func (s *posix) MakeVol(volume string) (err error) {
 	defer func() {
-		if err == syscall.EIO {
+		if isSysErrIO(err) {
```

Comment:
I think you mean here `if err == errFaultyDisk` because `isSysErrIO(errFaultyDisk)` should return false